### PR TITLE
{lib}[GCCcore/10.2.0] MATIO v1.5.21

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by robert-mijakovic** [comment]
@boegelbot: please test @ generoso

**Comment by robert-mijakovic** [comment]
Test report by @robert-mijakovic
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
login01.meluxina.lxp.lu - Linux RHEL 8.3, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/7367ef536306210c6555b97a15a39e1f for a full test report.

**Review by smoors — CHANGES_REQUESTED** [review]
@robert-mijakovic since the `autogen.sh` script runs `libtoolize` & co, you should probably add `Autotools` as build dep?

**Comment by robert-mijakovic** [comment]
> @robert-mijakovic since the `autogen.sh` script runs `libtoolize` & co, you should probably add `Autotools` as build dep?

Thanks for noticing. Done.

File: easybuild/easyconfigs/m/MATIO/MATIO-1.5.21-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+github_account = 'tbeu'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['2b09738fc765609794cdd4745649dcb2d7a528e9ef427c3b8e2a0d71d37d02f7']
+
+preconfigopts = './autogen.sh && '
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('Autotools', '20200128'),
```
Comment (verdurin) [inline easybuild/easyconfigs/m/MATIO/MATIO-1.5.21-GCCcore-10.2.0.eb:25]: I think this should be:
```suggestion
    ('Autotools', '20200321'),
```

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by branfosj** [comment]
Sorry, we missed following up on this and merged  #13710

**Pull request closed without merging**